app/app.go

```diff
@@ -157,6 +157,7 @@ import (
 	upgrade02 "github.com/sagaxyz/ssc/app/upgrades/0.2"
 	upgrade03 "github.com/sagaxyz/ssc/app/upgrades/0.3"
 	upgrade1 "github.com/sagaxyz/ssc/app/upgrades/1.0"
+	upgrade2 "github.com/sagaxyz/ssc/app/upgrades/2"
```

Comment:
You can call it anything but in keeping with the convention we have been following of naming the handler the same as the version of the build, this should be 1.2.